src/Tasks/SGen.cs

```diff
@@ -404,6 +414,8 @@ protected override string GenerateCommandLineCommands()
 
                 commandLineBuilder.AppendPlusOrMinusSwitch("/compiler:/delaysign", Bag, "DelaySign");
 
+                commandLineBuilder.AppendPlusOrMinusSwitch("/compiler:/publicsign", Bag, "PublicSign");
```

Comment:
Is there anything special that needs to be done here?

`publicsign+` and `delaysign+` can't be both set at the same time, but `publicsign+` and `delaysign-` can.

I'm assuming it would fail earlier in the build process.